x/gov/keeper/msg_server.go

```diff
@@ -31,6 +32,17 @@ func (k msgServer) SubmitProposal(goCtx context.Context, msg *types.MsgSubmitPro
 		return nil, err
 	}
 
+	bytes, err := proposal.Marshal()
+	if err != nil {
+		return nil, err
+	}
+
+	// ref: https://github.com/cosmos/cosmos-sdk/issues/9683
+	ctx.BlockGasMeter().ConsumeGas(
+		uint64(3*int(store.KVGasConfig().WriteCostPerByte)*len(bytes)),
+		"Submit proposal",
```

Comment:
```suggestion
		"submit proposal",
```

i believe lowercase is the convention elsewhere in the code